.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
remove this file from repo. add this folder to .gitignore

Reply:
ok sir i will change it